pages/linux/caffeinate.md

```diff
@@ -6,3 +6,7 @@
 - Prevent desktop from sleeping (use `Ctrl + C` to exit):
 
 `caffeinate`
+
+- Prevent disk from sleeping until you type Ctrl-C:
+
+`caffeinate -m`
```

Comment:
The linux command doesn't have this `-m` flag

Reply:
Sure. Will have a look

Reply:
I agree, on linux the command doesn't have the flag. Reverted the changes